composer.json

```diff
@@ -25,7 +25,7 @@
         "ext-zip": "*",
         "async-aws/ses": "^1.4",
         "bitbucket/client": "^4.0",
-        "buddy-works/buddy-works-php-api": "1.3.0",
```

Comment:
@akondas is there a specific reason to lock 1.3.0 before? We could use `^1.4` instead, which can be used to upgrade to any supported version in the 1x range.

Reply:
there is no specific reason, you can change this to `^1.4` :+1:

Reply:
I've changed this, more easy updates in the future 👍